packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -66,11 +66,6 @@ class PerformanceMonitoring {
       const payload = this.createTransactionPayload(tr)
       if (payload) {
         this._apmServer.addTransaction(payload)
-      } else if (__DEV__) {
-        this._logginService.debug(
-          'Could not create a payload from the Transaction',
-          tr
-        )
```

Comment:
Why is this no longer needed?

Reply:
We have a more specific log of why we are discarding it. Its just double log which feels unnecessary. 

We have both of these
1. we discard the transaction
2. could not create payload. 

Having just the 1st one feels clear and not polluting the logs as well.